.github/ISSUE_TEMPLATE/agents-discussion.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Agents discussion
+about: Creates a agent discussion issue
+labels: agents, discussion
+---
+
+<!--
+Once the discussion is at a s
+-->
```

Comment:
```suggestion
```

Reply:
I forgot to finish up that file. Added more context in the about section.